client/app/telecom/telephony/alias/configuration/translations/Messages_fr_FR.xml

```diff
@@ -38,4 +38,59 @@
+
+   <translation id="telephony_alias_config_change_type_label_conference" qtlid="333074">Conférence</translation>
+   <translation id="telephony_alias_config_change_type_desc_conference">La conférence vous permet d’être en communication téléphonique avec plusieurs interlocuteurs en simultané.</translation>
+
+   <translation id="telephony_alias_config_change_type_label_cloudIvr" qtlid="333100">Serveur Vocal Interactif</translation>
+   <translation id="telephony_alias_config_change_type_desc_cloudIvr">Un serveur vocal interactif vous permet de guider l’appelant vers le service adéquat via plusieurs choix à effectuer.</translation>
+
+   <translation id="telephony_alias_config_change_type_label_svi">SVI VXML</translation>
+   <translation id="telephony_alias_config_change_type_desc_svi">Le Serveur vocal interactif en VXML vous permet, via une configuration 2.1, d’utiliser un menu interactif avancé. L’appelant est invité, via des messages pré-enregistrés ou une synthèse vocale, à interagir avec le serveur grâce aux touches de son téléphone. Selon la configuration choisie, il sera possible de lire des sons ou de transférer un appel vers d’autres numéros. Notez que seuls les transferts vers les numéros OVH sont possibles.</translation>
+
+   <translation id="telephony_alias_config_change_type_label_easyHunting" qtlid="324457">File d'appels</translation>
+   <translation id="telephony_alias_config_change_type_desc_easyHunting" qtlid="333152">Avec la file d'appels, vous pouvez gérer le flux de vos appels.<br/><br/>Jouez un son d'accueil, personnalisez votre musique d'attente et faites sonner vos collaborateurs ou différents services en fonction de stratégies que vous définissez, sur des plages horaires choisies. Vous pouvez définir le nombre maximum d'appelants dans la file d'appels et le temps d'attente maximum. Si vous ne pouvez pas répondre, votre interlocuteur sera redirigé vers le répondeur d'une ligne OVH.<br/><br/><b class="text-danger">Veuillez noter que dans ce mode de fonctionnement, les renvois d'appels des membres de votre file d'appels seront automatiquement désactivés.</b></translation>
+
+   <translation id="telephony_alias_config_change_type_label_cloudHunting" qtlid="333165">File d'appels (mode expert)</translation>
+   <translation id="telephony_alias_config_change_type_desc_cloudHunting">Avec le mode expert de la file d’appels, vous pouvez gérer le flux de vos appels de bout en bout.<br/>Sur une interface similaire à celle du serveur vocal interactif, programmez étape par étape le déroulement de vos appels entrants.<br/>En fonction des numéros appelants ou de plages horaires, jouez des sons d’accueils, personnalisez votre musique d’attente, faites sonner vos collaborateurs ou différents services suivant des stratégies que vous définissez.<br/>Notez que dans ce mode de fonctionnement, les renvois d’appels des membres de votre file d’appels seront automatiquement désactivés.</translation>
```

Comment:
Could you split into multiple `<translation>` keys? (avoid `<br/>` tag I mean)

Reply:
In fact, this one can be deleted